internal/cmd/connect/connect.go

```diff
@@ -173,7 +175,7 @@ func runProxy(ctx context.Context, proxyOpts proxy.Options, database, branch str
 			return
 		}
 
-		fmt.Printf("Secure connection to database %s and branch %s is established!.\n\nLocal address to connect your application: %s (press ctrl-c to quit)\n",
+		fmt.Fprintf(color.Output, "Secure connection to database %s and branch %s is established!.\n\nLocal address to connect your application: %s (press ctrl-c to quit)\n",
```

Comment:
We should use ch.Printf which handles this. Looks like we forget to change it here. Thanks for the change though 👍